README.rst

```diff
@@ -1,64 +1,241 @@
 .. image:: https://coveralls.io/repos/zalando/lizzy/badge.svg?branch=master&service=github
    :target: https://coveralls.io/github/zalando/lizzy?branch=master
 
+=====
 Lizzy
 =====
 
-REST Service to deploy AWS Cloud Formation templates using `Senza`_
-CLI tool.
+Lizzy is a wrapper around `Senza`_ (a command line deployment tool to
+create and execute AWS CloudFormation templates) that allows cross
+AWS accounts deployments.
+
+Why Lizzy?
+==========
+
+Lizzy was created with the objective of enabling autonomous teams at
+`Zalando`_ to use Jenkins pipelines to continuously deploy new versions of
+their apps to their AWS accounts using the `Senza`_ CLI tool.
+
+The managed Jenkinses at Zalando run under the Continuous Delivery team's
+AWS account. The Continuous Delivery team is responsible for
+maintaining the Jenkinses of all other Zalando teams. That allows
+other teams in the company to focus on their `OKR's`_ and not invest
+their time on configuring Jenkins or continuous deployment tools. The
+teams at Zalando enjoy `Jenkins-as-a-Service`_.
+
+Each team at Zalando has their own AWS account. So running the
+`Senza`_ tool within a managed Jenkins job would allow deployments only in
+the Continuous Delivery team's AWS account. To permit different teams
```

Comment:
> So running the `Senza`_ tool within a managed Jenkins job would allow deployments only in the Continuous Delivery team's AWS account.

It's not clear to me what you are trying to convey here, I am afraid. 
Do you mean to say that Senza tool runs managed Jenkins jobs in CD team's AWS account?